angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -38,14 +39,7 @@ export class CustomAppComponent {
     public activeAnchorTabId = 'a-tab-2';
 
     public readonly tableData$: Observable<SimpleTableRecord[]>;
-    private readonly tableDataSubject = new BehaviorSubject<SimpleTableRecord[]>([
-        { stringValue1: 'hello world', stringValue2: 'more text' },
-        { stringValue1: 'foo', stringValue2: 'bar' },
-        { stringValue1: 'candy', stringValue2: 'bar' },
-        { stringValue1: 'dive', stringValue2: 'bar' },
-        { stringValue1: 're', stringValue2: 'bar' },
-        { stringValue1: 'last row', stringValue2: 'yay!' }
-    ]);
+    private readonly tableDataSubject = new BehaviorSubject<SimpleTableRecord[]>([]);
```

Comment:
If we're fine with this change, I can create an issue for adding initial table data once the accessibility concerns are addressed.

Reply:
Created #1195